cddagl/ui.py

```diff
@@ -122,11 +161,64 @@ def arstrip(value):
         value = value[:-1]
     return value
 
-def is_64_windows():
-    return 'PROGRAMFILES(X86)' in os.environ
+def is_osx():
+    return platform.system() == 'Darwin'
+
+def is_windows():
+    return platform.system() == 'Windows'
+
+def is_linux():
+    return platform.system() == 'Linux'
+
+def is_posix():
+    return os.name == 'posix'
+
+def is_64_system():
+    return sys.maxsize > 2**32

```

Comment:
Agree, platform.architecture()[0] gives more reliable result according to http://stackoverflow.com/questions/1842544/how-do-i-detect-if-python-is-running-as-a-64-bit-application